From 03580853b308ab309433d4fc74ccbeb2f212b0c9 Mon Sep 17 00:00:00 2001
From: Marcelo Leitner <mleitner@redhat.com>
Date: Tue, 21 Oct 2014 16:16:18 -0400
Subject: [net] nf_tables: Include appropriate header file in netfilter/nft_lookup.c

Message-id: <20141021161618.22888.13160.stgit@localhost.localdomain>
Patchwork-id: 97912
O-Subject: [RHEL7 net PATCH 015/124] net: Include appropriate header file in netfilter/nft_lookup.c
Bugzilla: 1148041 1155088
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

Backport of upstream commit: bd76ed36bac50b5402fd09c3fc2f368ef324ffe2

 commit bd76ed36bac50b5402fd09c3fc2f368ef324ffe2
 Author: Rashika Kheria <rashika.kheria@gmail.com>
 Date:   Sun Feb 9 22:31:42 2014 +0530

    net: Include appropriate header file in netfilter/nft_lookup.c

    Include appropriate header file net/netfilter/nf_tables_core.h in
    net/netfilter/nft_lookup.c because it has prototype declaration of
    functions defined in net/netfilter/nft_lookup.c.

    This eliminates the following warning in net/netfilter/nft_lookup.c:
    net/netfilter/nft_lookup.c:133:12: warning: no previous prototype for 'nft_lookup_module_init' [-Wmissing-prototypes]
    net/netfilter/nft_lookup.c:138:6: warning: no previous prototype for 'nft_lookup_module_exit' [-Wmissing-prototypes]

    Signed-off-by: Rashika Kheria <rashika.kheria@gmail.com>
    Reviewed-by: Josh Triplett <josh@joshtriplett.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Marcelo Ricardo Leitner <mleitner@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/netfilter/nft_lookup.c b/net/netfilter/nft_lookup.c
index 8a6116b..bb4ef4c 100644
--- a/net/netfilter/nft_lookup.c
+++ b/net/netfilter/nft_lookup.c
@@ -16,6 +16,7 @@
 #include <linux/netfilter.h>
 #include <linux/netfilter/nf_tables.h>
 #include <net/netfilter/nf_tables.h>
+#include <net/netfilter/nf_tables_core.h>
 
 struct nft_lookup {
  struct nft_set   *set;
-- 
1.7.1